### PR TITLE
Update release process notes for PyPI

### PR DESCRIPTION
**Pull request by holdenk**
Update the release process notes for PyPI uploads.

### CODE REVIEW

**Review by srowen — APPROVED** [review]

File: release-process.md [hunk 1]
```diff
@@ -130,6 +130,19 @@ Verify that the resources are present in <a href="http://www.apache.org/dist/spa
 It may take a while for them to be visible. This will be mirrored throughout the Apache network. 
 There are a few remaining steps.
 
+<h4>Upload to PyPI</h4>
+
+Uploading to PyPI is done after the release has been uploaded to apache. To get started, go to the <a href="https://pypi.python.org">PyPI website</a> and log in with the spark-upload account (see the PMC mailing list for account permissions).
```
Comment (srowen) [inline release-process.md]: Actually, one nit: apache -> Apache

File: release-process.md [hunk 2]
```diff
@@ -130,6 +130,19 @@ Verify that the resources are present in <a href="http://www.apache.org/dist/spa
...
+
+
+Once you have logged in it is time to register the new release, on the <a href="https://pypi.python.org/pypi?%3Aaction=submit_form">submitting package information</a> page by uploading the PKG-INFO file from inside the pyspark packaged artifact.
+
+Once the release has been registered you can upload the artifacts on the <a href="https://pypi.python.org/pypi?%3Aaction=pkg_edit&name=pyspark"> pyspark package edit page</a>.
```
Comment (srowen) [inline release-process.md]: pyspark -> PySpark or is this on purpose to refer to part of a file name?

**Comment by srowen** [comment]
@holdenk not sure if my reply reached you on this one

**Comment by holdenk** [comment]
Just noticed it. I'll hold off on merging this once we've got the version published to PyPi just incase there are any steps hiding. Will fix the rest tomorrow :)

**Comment by srowen** [comment]
@holdenk do you want this merged or does it need to wait?

**Comment by holdenk** [comment]
I think this should wait until 2.2 is released and I can confirm for sure that there aren't any other weird pieces we need to do as part of being a larger PyPI package.

**Comment by srowen** [comment]
@holdenk looks like PyPI was a success -- should I merge this?

**Comment by holdenk** [comment]
Let me update it tonight, I found something during the release process we should document incase anyone else runs into the same problem as I do during the upload.

File: release-process.md [hunk 3]
```diff
@@ -132,6 +132,23 @@ Verify that the resources are present in <a href="https://www.apache.org/dist/sp
...
 
+<h4>Upload to PyPI</h4>
+
+Uploading to PyPI is done after the release has been uploaded to Apache. To get started, go to the <a href="https://pypi.python.org">PyPI website</a> and log in with the spark-upload account (see the PMC mailing list for account permissions).
+
+
+Once you have logged in it is time to register the new release, on the <a href="https://pypi.python.org/pypi?%3Aaction=submit_form">submitting package information</a> page by uploading the PKG-INFO file from inside the pyspark packaged artifact.
+
+
+Once the release has been registered you can upload the artifacts on the
```
Comment (cloud-fan) [inline release-process.md]: remove "on the"?

**Comment by cloud-fan** [comment]
is it ready to go?

File: release-process.md (hunk 3 above)
Comment (holdenk) [reply release-process.md]: oh good point. give me a second.

**Comment by srowen** [comment]
@holdenk feel free to merge if https://issues.apache.org/jira/browse/SPARK-20676 is done

**Comment by holdenk** [comment]
Merged to asf-site

**Pull request closed without merging**